modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -46,19 +43,150 @@ Find the {minimalsles} image at https://download.suse.com/.
 For more information on {sle} Virtualization Guide, see https://documentation.suse.com/sles/15-SP4/html/SLES-all/book-virtualization.html.
 ====
 
+[IMPORTANT]
+====
+{productname} VM image does not setup [systemitem]``root`` or any other user account. User or [systemitem]``root`` authentications need to be setup during first boot. This can be done using [systemitem]``Ignition`` or [systemitem]``Cloud-Init`` methods.
```

Comment:
```suggestion
{productname} VM image does not setup [systemitem]``root`` or any other user account.
User or [systemitem]``root`` authentications need to be setup during first boot.
This can be done using [systemitem]``Ignition`` or [systemitem]``Cloud-Init`` methods.
```